src/assets/icons/iconMap.js

```diff
@@ -0,0 +1,349 @@
+import { ReactComponent as Aa } from './aa.svg'
+import { ReactComponent as AccountBreakdown } from './account-breakdown.svg'
+import { ReactComponent as AccountMlp } from './account-mlp.svg'
+import { ReactComponent as ActivityClipboard } from './activity-clipboard.svg'
+import { ReactComponent as AddOns } from './add-ons.svg'
+import { ReactComponent as Addons } from './addons.svg'
```

Comment:
What's the difference between these two different svgs?

Reply:
Apparently one has some pink and the other doesn't 😁

Reply:
Good to know!